### PR TITLE
Use CRA (Redo): Pt 1/3: Working "build" (not tests)

### PR DESCRIPTION
**Pull request by jessie-ross**
**This PR switches out NextJS for CRA** 
- Moves `/pages-real` to `/pages`
- Fixes storybook

## How has this been tested?

AT will be fixed in the next two PRs, but this can be tested by running `npm run dev`

### CODE REVIEW

**Comment by jessie-ross** [comment]
Vercel is configured for NextJs so it would need to have it's config changes (it has a preset for CRA).

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Web site created using create-react-app" />
```
Comment (yanirs) [inline public/index.html:8]: Maybe delete/change?

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (yanirs) [inline public/index.html:31]: Change to Abacus?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (yanirs) [inline public/index.html:45]: We can just delete this.
```suggestion
```

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (yanirs) [inline public/index.html:5]: We should probably add a proper favicon (not for this PR, unless you want to) :slightly_smiling_face:

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+    />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (yanirs) [inline public/index.html:21]: Should we delete this and the logo192.png ref?

File: src/react-app-env.d.ts [hunk 6]
```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```
Comment (yanirs) [inline src/react-app-env.d.ts:1]: Is this file needed?

**Review by yanirs — APPROVED** [review]
Looks good, except for a few minor/optional comments.

File: src/react-app-env.d.ts (hunk 6 above)
Comment (jessie-ross) [reply src/react-app-env.d.ts:1]: https://github.com/facebook/create-react-app/issues/6560#issuecomment-503682309

**Comment by jessie-ross** [comment]
Merged in #390
